core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1339,41 +1341,45 @@ public void onError(final Status error) {
       checkArgument(!error.isOk(), "the error status must not be OK");
       logger.log(Level.WARNING, "[{0}] Failed to resolve name. status={1}",
```

Comment:
I would move this line into the runnable to be consistent with the logging in `onAddresses()`.

Reply:
Done.